tests/test_bilstm.py

```diff
@@ -46,6 +46,27 @@ def test_save_load():
         assert hasattr(loaded_model, 'model')
         assert loaded_model.score(X_vec, Y) > 0.6
 
+def test_save_load_attention():
+    X = [
+        "One",
+        "One only",
+        "Two nothing else",
+        "Two and three"
+    ]
+    Y = np.array([0, 0, 1, 1])
+
+    vec = KerasVectorizer()
+    X_vec = vec.fit_transform(X)
+
+    model = BiLSTMClassifier()
+    model.fit(X_vec, Y)
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        model.save(tmp_dir)
+        loaded_model = BiLSTMClassifier(attention=True)
```

Comment:
should `attention=True` be added to like 61 rather than here? i.e. arent you testing whether you can load a model with attention?

Reply:
oups, good point

Reply:
5a7a87138b8a20a7bcda9b2518021ef06bf6ace1